### PR TITLE
Mes 3534 config schema

### PR DESCRIPTION
**Pull request by szabolcs-keresztes**
- Creating mes-config-schema package
- Adding `app-config` json schema
- Generating `app-config.d.ts`

### CODE REVIEW

File: mes-config-schema/app-config.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    "examinerRole": {
+      "description": "Role of the user who uses the the App",
+      "type": "string",
+      "enum": [
+        "LDTM",
+        "DE"
+      ]
+    },
+    "examinerRoleDescription": {
+      "description": "Description of theRole of the user who uses the the App",
```
Comment (rossfauldsbjss) [inline mes-config-schema/app-config.json]: theRole instead of the Role. (very, very, very minor)

Comment (szabolcs-keresztes) [reply mes-config-schema/app-config.json]: Fixed it

File: mes-config-schema/app-config.d.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+   */
+  category: string;
+  /**
+   * Date when the permission is active from
+   */
+  from: string;
+  /**
+   * Date till the permission is active
+   */
+  to?: string;
```
Comment (ammarhaiderbjss) [inline mes-config-schema/app-config.d.ts]: Are we sure this is optional?

**Comment by ammarhaiderbjss** [comment]
@szabolcs-keresztes will there be a separate schema for MDM config coming? In which case we should probably think about how to name them, perhaps `mes-remote-config-schema` and `mes-mdm-config-schema`?

File: mes-config-schema/app-config.d.ts (hunk 2 above)
Comment (szabolcs-keresztes) [reply mes-config-schema/app-config.d.ts]: Yeah, if not defined that means that it's indefinite

**Pull request merged**